src/Tabs/index.jsx

```diff
@@ -1,10 +1,51 @@
-import Tabs from 'react-bootstrap/Tabs';
+import React from 'react';
+import BaseTabs from 'react-bootstrap/Tabs';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 import TabsDeprecated from './deprecated';
+import Bubble from '../Bubble';
+
+const Tabs = ({
```

Comment:
[suggestion] @monteri Curious about the approach to modify the wrapper `Tabs` component to conditionally render the `Bubble` here versus extending the `Tab` component from react-bootstrap to reduce some of this complexity, keeping the change solely to `Tab`. Would something like the below work as well?

```jsx
import BaseTab from 'react-bootstrap/Tab';

const Tabs = React.forwardRef(({ notification, title, children, ...rest }, ref) => {
  let titleProp = title;
  if (notification) {
    titleProp = (
      <>
        {title}
        <Bubble variant="error" className="pgn__tab-notification">{notification}</Bubble>
      </>
    );
  }

  return (
    <BaseTab
      title={titleProp}
      {...rest}
    >
      {children}
    </BaseTab>
  );
});
```

Reply:
It was my first try and it was unsuccessful because of how `Tabs` component from `react-bootstrap` processes children. It doesn't render `Tab` component it uses only its props. So I think the only solution is via `Tabs` component then.

[Link to the source code for the tabs generating](https://github.com/react-bootstrap/react-bootstrap/blob/e32c53d9001e07f5685c701a41f0c0b7d5a39cca/src/Tabs.tsx#L104)

Reply:
Oh interesting... yeah, I see what you mean now; we will have to do this via `Tabs` as you said.